src/textfield/textfield.js

```diff
@@ -213,12 +213,33 @@
    * @public
    */
   MaterialTextfield.prototype.change = function(value) {
-
     this.input_.value = value || '';
     this.updateClasses_();
   };
   MaterialTextfield.prototype['change'] = MaterialTextfield.prototype.change;
 
+   /**
+   * Focus text field.
```

Comment:
JSCS is being picky and complaining about the indenting in these comments. Could you change these to be:
```js
/**
 * Focus text field.
 * [...]
```
instead?